samila/params.py

```diff
@@ -39,8 +39,11 @@
 CONFIG_NO_STR_FUNCTION_ERROR = "Config file can't be saved. At least one of the function1_str or function2_str is None."
 PLOT_DATA_ERROR = "Plotting process can't be Done because data{0} is empty. Use generate method first."
 SAVE_NO_DATA_ERROR = "Data file can't be saved. At least one of the data1 or data2 is None."
+INVALID_COLOR_TYPE_ERROR = "Given color type is not supported. Replacing it with the default color."
```

Comment:
Sometimes it's not the default color, it's the previously selected color!

Reply:
My suggestion:

Given color type is not supported. It is set to the previously selected or default one.